libs/ts-rest/nest/src/lib/api.decorator.ts

```diff
@@ -115,18 +116,38 @@ const getMethodDecorator = (appRoute: AppRoute) => {
 
 @Injectable()
 export class ApiRouteInterceptor implements NestInterceptor {
+  constructor(private readonly appRoute: AppRoute) {}
```

Comment:
Can we make this constructor accept a param to mark this feature as optional? Similar to what @Gabrola suggested.

Reply:
We shouldn't pass values manually to injectable class constructors because the DI container would fail to instantiate new instances if needed. The NestJS way would be to create a custom decorator for example `@ValidateResponses()`, and use metadata to check if it's enabled.